src/octree/raytracing/raytracing_on_cpu.rs

```diff
@@ -162,7 +163,7 @@ impl<T: Default + PartialEq + Clone + std::fmt::Debug + VoxelData, const DIM: us
                 current_index.z as usize,
                 DIM,
                 matrix_occupied_bits,
-            ) || !matrix[current_index.x as usize][current_index.y as usize]
+            ) && !matrix[current_index.x as usize][current_index.y as usize]
```

Comment:
oh, wow... that's a nasty one

Reply:
yea, but it only had the effect of the bitmap stuff not working, so it took a while to figure out